.github/workflows/publish-preview.yml

```diff
@@ -7,37 +7,18 @@ on:
       - release-*
 
 jobs:
-  filter:
-    name: Confirm NPM_AUTH_TOKEN Access
+  preview: 
+    name: Publish Preview Package
     runs-on: ubuntu-latest
     steps:
-    - name: Confirm NPM_AUTH_TOKEN Access
-      uses: thefrontside/actions/merge-and-fork-detector@master
-      with:
-        NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
-
-  publish: 
-    name: Publish Commit Package
-    runs-on: ubuntu-latest
-    needs: filter
-    steps:
     - uses: actions/checkout@v1
     - name: Write NPM Snapshot Version
       uses: thefrontside/actions/write-npm-snapshot-version@master
     - name: NPM Publish Commit
-      uses: actions/npm@master
-      with: 
-        args: publish --access=public --tag dev
+      uses: thefrontside/actions/npm-publish-branch-preview@master
       env:
         NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
-
-  comment: 
-    name: Post Instruction Comment
-    runs-on: ubuntu-latest
-    needs: publish
-    steps:
-    - uses: actions/checkout@v1
-    - name: Post Comment
+    - name: Post Instructions Comment
       uses: thefrontside/actions/post-npm-usage-instructions-comment@master
-      env: 
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+      env:
+        GITHUB_TOKEN: ${{ secrets.POST_COMMENT_TOKEN }}
```

Comment:
Let's call this `GITHUB_POST_COMMENT_TOKEN` so it's completely unambiguous what service this is for

Reply:
Done